srcpkgs/dotnet-runtime/template

```diff
@@ -0,0 +1,42 @@
+# Template file for 'dotnet-runtime'
+pkgname=dotnet-runtime
+version=2.2.4
+revision=1
+archs="x86_64 armv7 aarch64"
+wrksrc="${pkgname}-${version}"
+create_wrksrc=yes
+short_desc="Provides the .NET core runtime components"
+maintainer="Anachron <gith@cron.world>"
+license="MIT"
+homepage="https://dotnet.microsoft.com"
+conflicts="dotnet-sdk"
+
+case "${XBPS_TARGET_MACHINE}" in
+	x86_64)
+	_arch="x64"
+	_path="853048a3-764a-4b4d-a608-c6144a84f257/99c5cb1ea145f9dc3c2bbd093c682c9b"
+	distfiles="https://download.visualstudio.microsoft.com/download/pr/${_path}/${pkgname}-${version}-linux-${_arch}.tar.gz"
+	checksum="0e494df7a3936ac59c17de3b91d928bb3ab3cdd1e6734d581ad4774f551ca239"
+	;;
+	armv7)
+	_arch="arm"
+	_path="8c52648c-bedd-44b0-9442-95cd830fdada/d6ba4c50a6b2afddc4ae3d313349f3ac"
+	distfiles="https://download.visualstudio.microsoft.com/download/pr/${_path}/${pkgname}-${version}-linux-${_arch}.tar.gz"
+	checksum="06cc0010e92591c350fe010feacdd6bb55294f89e97ea30eac5a46c33fd8d1f4"
+	;;
+	aarch64)
+	_arch="arm64"
+	_path="9e06922d-3a96-4f0d-9eb0-94f2cf94458f/93dfe5f0ad50c0eb347e98d7f81b34ec"
+	distfiles="https://download.visualstudio.microsoft.com/download/pr/${_path}/${pkgname}-${version}-linux-${_arch}.tar.gz"
+	checksum="492de061d1e01862c2f208287c5454a19650ab2d6441554347d847656900da70"
+	;;
+esac
+
+nopie=yes
+nopie_files="/usr/bin/dotnet"
```

Comment:
Isn't that redundant? Afair `nopie_files` is there to mark binaries which are not compiled as pie in packages that otherwise contain pie-files. If the whole package is marked nopie, I see no reason to also use `nopie_files`.